# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/LocalAccessible.h
# Commit: 7b00a16ad21e
# Full Hash: 7b00a16ad21ee2b820b679199ae3806250d861e9
# Author: Nathan LaPre <nlapre@mozilla.com>
# Date: 2024-09-10 09:03:27
# Regressor Bug: 1794974
# File Overlap Count: 1
# Description:
#   Bug 1794974: Part 2: Add ability to query and send individual cache domains, r=Jamie
#   
#   This revision adds the capability of querying and sending information about
#   individual cache domains. It introduces the concept of active cache domains to
#   the accessibility service: cache domains that we think clients need. Conversely,
# ==============================================================================

diff -r 642b6d8c4d3b -r 7b00a16ad21e accessible/generic/LocalAccessible.h
--- a/accessible/generic/LocalAccessible.h	Mon Sep 09 23:02:19 2024 +0000
+++ b/accessible/generic/LocalAccessible.h	Mon Sep 09 23:02:19 2024 +0000
@@ -9,6 +9,7 @@
 #include "mozilla/ComputedStyle.h"
 #include "mozilla/a11y/Accessible.h"
 #include "mozilla/a11y/AccTypes.h"
+#include "mozilla/a11y/CacheConstants.h"
 #include "mozilla/a11y/RelationType.h"
 #include "mozilla/a11y/States.h"
 
@@ -715,7 +716,8 @@
   virtual bool IsRemote() const override { return false; }
 
   already_AddRefed<AccAttributes> BundleFieldsForCache(
-      uint64_t aCacheDomain, CacheUpdateType aUpdateType);
+      uint64_t aCacheDomain, CacheUpdateType aUpdateType,
+      uint64_t aInitialDomains = CacheDomain::None);
 
   /**
    * Push fields to cache.